eyre/tests/test_location.rs

```diff
@@ -122,6 +122,7 @@ fn test_with_context() {
         Box::new(LocationHandler::new(expected_location))
     }));
 
+    use eyre::WrapErr;
```

Comment:
<img width="1894" alt="Screenshot 2024-01-24 at 17 30 21" src="https://github.com/eyre-rs/eyre/assets/25723553/e27daaf6-824c-4155-9f27-706a60b85cf3">

Github decides to conveniently fold this code to make it appear as if it is in a different function.

This is what lead to some of my initial confusion in the #138 PR until I spotted the little code fold when scanning upwards for the function name 😅 

Anyhow, nice clarification on the traits, makes in-browser reviewing easier, and I am working on a PR to potentially fix this entirely.

Reply:
![image](https://github.com/eyre-rs/eyre/assets/22329650/bd7adcac-684b-4742-b49e-005e87b427b0)

"Expand all" can be helpful in diffs of rather uniform code, such as structurally repetitive tests.